tests_and_analysis/test/euphonic_test/test_force_constants_calculate_qpoint_frequencies.py

```diff
@@ -37,6 +37,10 @@ def get_quartz_fc():
         (get_quartz_fc(), 'quartz',
          [get_test_qpts(), {'asr': 'reciprocal', 'splitting': False}],
          'quartz_reciprocal_qpoint_frequencies.json'),
+        (get_quartz_fc(), 'quartz',
+         [get_test_qpts(), {'asr': 'reciprocal', 'splitting': False,
+                            'dipole_parameter': 0.75}],
+         'quartz_reciprocal_qpoint_frequencies.json'),
```

Comment:
This looks like some duplication above and below. Also the lines below still have eta_scale in them

Reply:
I feel like it's good practice to keep testing `eta_scale` even though its deprecated until it's been removed, right? Unless deprecation also means, 'not testing anymore, use at your own risk'?

There is a lot of duplication in testing `QpointPhononModes/Frequencies` as they essentially behave the same and use the same underlying code, except one doesn't contain any eigenvectors. It's probably still a good idea to explicitly test both of them though in case we break passing through the arguments through, for example.